azure-pipelines.yml

```diff
@@ -0,0 +1,156 @@
+# azure-pipelines build file
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema
+
+# TODO: Add gh-pages deployment on tag trigger
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - RB-*
+    - ci_test
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - master
+    - RB-*
+
+# Common variables
+variables:
+  cxxStd: 14
+  nixCxxFlags: -std=c++$(cxxStd) -fPIC
+
+jobs:
+
+# ------------------------------------------------------------------------------
+# Linux
+# ------------------------------------------------------------------------------
+# TODO: Move Docker images to ASWF DockerHub account
+
+- job: Linux
+  strategy:
+    matrix:
+      Ubuntu 18.04 VFX CY2020:
+        agentImage: 'ubuntu-16.04'
+        containerImage: michdolan/ubuntu18.04:vfx2020_py27
+        cxxFlags: ''
```

Comment:
There is some confusion here: the name refers to 18, but the image is 16, and the container is 18.

Reply:
In this case, the 16 agentImage is just the host machine, but the build is done in the Docker container which is 18. Azure only offers Ubuntu 16 for Linux builds, but the Docker container fully insulates from that, so we can do builds in CentOS, or whatever other Linux distro we want.

Reply:
ok. Please add a one-line explanation for readability.